R/MetaboLightsParam.R

```diff
@@ -0,0 +1,72 @@
+#' @title Load content from a MetaboLights study
+#'
+#' @name MetaboLightsParam
+#'
+#' @export
+#'
+#' @family MS object export and import formats
+#'
+#' @description
+#'
+#' The `MetaboLightsParam` class and the associated `readMsObject()` method
+#' allow users to load an `MsExperiment` object from a study in the
+#' MetaboLights database (https://www.ebi.ac.uk/metabolights/index) by
+#' providing its unique study `studyId`. This function is particularly useful
+#' for importing metabolomics data into an `MsExperiment` object for further
+#' analysis within the R environment.
```

Comment:
Maybe specify that at present we don't support a `writeMsObject()` method for this param class, i.e. we can only import, but not export.